### DISCUSSION TITLE
[Offtop] Supporting Supabase

### DISCUSSION THREAD

**Discussion by n-sviridenko** [AntlerVC/firetable#806] [Support / Q&A] [answered]
Maybe someone knows about existing solutions? Or maybe you guys plan to have something in this direction?

**Comment by shamsmosowi** [comment, marked as answer]
Hi @n-sviridenko !, We're exploring Postgres support  atm.
I am aware that Supabase provides a builtin CMS, Interested to know which parts of Rowy you miss the most?

**Comment by n-sviridenko** [comment]
This CMS is for the admin users having access to all data (like the Firebase Admin Console), while we want to have a rowy-like access (through supabase auth, analog of firebase auth).

**Comment by n-sviridenko** [comment]
I think supporting postgres directly doesn't make sense, as we have nocodb for that, who've done a lot of stuff already. So supporting supabase will bring a bigger benefit, as they're no solutions on the market for this problem atm.
